timeline-chart/src/layer/time-graph-layer.ts

```diff
@@ -48,10 +48,12 @@ export abstract class TimeGraphLayer {
 
     protected removeChildren() {
         this.children.forEach(child => this.layer.removeChild(child.displayObject));
+        this.children.forEach(child => child.destroy());
         this.children = [];
     }
 
     protected removeChild(child: TimeGraphComponent) {
+        child.destroy();
```

Comment:
this will destroy the child's displayObject. However, it is used later on in line 59. I wonder if it should be removed first, before destroyed? 

I debugged the removeChild(child.displayObject)  with your implementation and after moving the destroy after the removeChild() line. I noticed that inside the method it takes a different code path when stepping through.

Reply:
I checked the documentation and it seems that calling destroy() on a DisplayObject will automatically remove it from its parent Container. (https://pixijs.download/release/docs/PIXI.DisplayObject.html#destroy). So maybe I should update the patch and remove the this.layer.removeChild() which is no longer needed ?

Reply:
> I checked the documentation and it seems that calling destroy() on a DisplayObject will automatically remove it from its parent Container. (https://pixijs.download/release/docs/PIXI.DisplayObject.html#destroy). So maybe I should update the patch and remove the this.layer.removeChild() which is no longer needed ?

Yes, this will work too. Thanks for checking.